app/assets/stylesheets/partials/custom.scss

```diff
@@ -181,7 +182,7 @@ background-color: #fff !important;
 .servive-block-light,
 .servive-block-default {
   background: #fafafa;
-  border: solid 1px #eee; 
+  border: solid 1px #eee;
```

Comment:
Color literals like `#eee` should only be used in variable declarations; they should be referred to via variable everywhere else.